great_expectations/data_context/store/configuration_store.py

```diff
@@ -112,6 +113,10 @@ def deserialize(self, value):
         except gx_exceptions.InvalidBaseYamlConfigError:
             # Just to be explicit about what we intended to catch
             raise
+        except marshmallow.ValidationError as e:
```

Comment:
Still looking, but I wouldn't have expected this change. Why is this coming up in this PR?